script-library/common-debian.sh

```diff
@@ -70,6 +70,8 @@ apt-get-update-if-needed()
 
 # Run install apt-utils to avoid debconf warning then verify presence of other common developer tools and dependencies
 if [ "${PACKAGES_ALREADY_INSTALLED}" != "true" ]; then
+    # Adding non-free packges on Debian
```

Comment:
Can you explain why we need to do this in the comment (needed for the manages packages)?

Reply:
Explained in the comment